docs/using-modules.rst

```diff
@@ -190,5 +197,11 @@ So, exporting external functions from modules is accomplished using the ``export
         ownable_2step.accept_ownership,
     )
 
+    # export all functions defined in IERC20
+    exports: token.IERC20
+
     # export all external functions from `ownable_2step`
     exports: ownable_2step.__interface__
+
+.. note::
+    Any exported interfaces must be implemented by the module. For example, in the above example, ``token`` must ``implement`` ``IERC20``, or else the compiler will raise an error.
```

Comment:
Is `token` a typo here? Seems to be an import but then you are referring to it like it is the target

Reply:
not a typo -- it must be implemented in the imported module. we could maybe call it `base_token`?

Reply:
37913779f5f1ec132f2e085c2b3611c6c6ce73d1, does this help?